spark-rapids/spark-rapids.sh

```diff
@@ -340,6 +340,16 @@ EOF
 readonly NVIDIA_BASE_DL_URL='https://developer.download.nvidia.com/compute'
 readonly NVIDIA_REPO_URL="${NVIDIA_BASE_DL_URL}/cuda/repos/${shortname}/x86_64"
 
+# Hold all NVIDIA-related packages from upgrading unintenionally or services like unattended-upgrades
+# Users should run apt-mark unhold before they wish to upgrade these packages
+function hold_nvidia_packages() {
+  apt-mark hold nvidia-*
+  apt-mark hold libnvidia-*
+  if apt list --installed 2>/dev/null | grep -q "xserver-xorg-video-nvidia"; then
```

Comment:
`dpkg -l` is lighter weight than `apt list --installed` ; I prefer not to use 'apt' and instead prefer 'apt-get' when it must be used in automation.  The ABI of apt is not stable.

Reply:
Updated, thanks!